code/datums/mind.dm

```diff
@@ -1172,7 +1172,6 @@
 				log_admin("[key_name(usr)] has devil'ed [current]. The devil has been marked as ascendable.")
 			if("sintouched")
 				if(ishuman(current))
-					ticker.mode.sintouched += src

```

Comment:
Okay then I mis remembered then but there is a way to only add if it isnt in there already correct?